0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -133,9 +133,8 @@ Retrieve some metadata about the ledger. Plugin must be connected, otherwise the
 {
   "prefix": "us.fed.some-bank.",
   "precision": 10,
```

Comment:
Why are we changing scale to currencyScale, but not changing precision to currencyPrecision? I'm not so sure I agree with the renaming in any case, because the scale and precision are ledger properties, not currency properties. `currencyCode` is a description of the ledger's currency code, not a description of the ledger's currency, if that makes sense. Something like `amountScale` and `amountPrecision` might make a little more sense, but I think might be more verbose than we need.